packages/fluent-theme/src/fluent/styles/SearchBox.styles.ts

```diff
@@ -51,13 +51,13 @@ export const SearchBoxStyles = (props: ISearchBoxStyleProps): Partial<ISearchBox
       padding: 0,
       selectors: {
         '&:hover .ms-Button': {
-          backgroundColor: palette.neutralLighter
+          backgroundColor: semanticColors.buttonBackground
         },
         '&:hover .ms-Button-icon': {
-          color: palette.neutralPrimary
+          color: semanticColors.buttonText
         },
         '.ms-Button-icon': {
-          color: palette.neutralSecondary
+          color: semanticColors.inputPlaceholderText
```

Comment:
not sure if this slot is used correctly here. Although the color matches and you might say it's a placeholder. This is why I say we need designers to provide us redlines on slots usage for every component.

Reply:
@Vitalius1 would semanticColors.bodySubtext make more sense for this scenario or change back to neutralSecondary

Reply:
I think `bodySubtext` make more sense in here as long a it's `neutralSecondary` mapped. @phkuo is the expert I would recommend when in doubt what to choose 😄

Reply:
not sure without looking at redlines, but see my comments elsewhere. you should start by picking a category of semantic slots to use, and use the appropriate states from there.